test/regression.uts

```diff
@@ -2156,13 +2157,12 @@ pktcapng = sniff(offline=scapy_path("/test/pcaps/macos.pcapng.gz"))
 assert len(pktcapng) != 0
 
 = Write a pcapng
-
```

Comment:
Please refrain from adding un-necessary changes.

Reply:
thanks for feedback! I will try avoid it in future.